core/client/bower.json

```diff
@@ -20,14 +20,14 @@
     "jquery-file-upload": "9.5.6",
     "jquery-hammerjs": "1.0.1",
     "jquery-ui": "1.11.4",
-    "jqueryui-touch-punch": "furf/jquery-ui-touch-punch",
+    "jqueryui-touch-punch": "furf/jquery-ui-touch-punch#4bc009145202d9c7483ba85f3a236a8f3470354d",
     "jquery.simulate.drag-sortable": "0.1.0",
     "keymaster": "1.6.3",
     "loader.js": "3.2.1",
     "lodash": "~3.7.0",
     "moment": "2.10.3",
     "normalize.css": "3.0.3",
-    "password-generator": "git://github.com/bermi/password-generator#49accd7",
+    "password-generator": "git://github.com/bermi/password-generator#49accd739f60dfd0770d4af231f38311f482e308",
```

Comment:
This could just be switched to "1.0.0" because that's the release the closest to this commit.